bundles/org.openhab.ui/web/src/assets/openhab-js-tern-defs.json

```diff
@@ -201,6 +293,15 @@
                 "!type": "fn(key: string) -> ?"
             }
         },
+        "TimeNamespace": {
+            "toZDT": {
+                "!doc": "Converts nearly any representation of a time to a time.ZonedDateTime (see JS Scripting Docs/Standard Library/Time)",
+                "!type": "fn(time: any) -> time.ZonedDateTime"
```

Comment:
Please note that `any` is a TypeScript keyword, here is should be `?`.

Reply:
Okay thanks.
I will do this ASAP.